examples/kubernetes/1.10/cilium-ds.yaml

```diff
@@ -35,7 +35,7 @@ spec:
         - name: clean-cilium-state
           image: docker.io/library/busybox:1.28.4
           imagePullPolicy: IfNotPresent
-          command: ['sh', '-c', 'if [ "${CLEAN_CILIUM_STATE}" = "true" ]; then rm -rf /var/run/cilium/state; fi; if [ "${CLEAN_CILIUM_STATE}" = "true" ] || [ "${CLEAN_CILIUM_BPF_STATE}" = "true" ]; then rm -rf /sys/fs/bpf/tc/globals/cilium_* /var/run/cilium/bpffs/tc/globals/cilium_*; fi']
+          command: ['sh', '-c', 'rm /var/run/cilium/{cilium.pid,state/health-endpoint.pid}; if [ "${CLEAN_CILIUM_STATE}" = "true" ]; then rm -rf /var/run/cilium/state; fi; if [ "${CLEAN_CILIUM_STATE}" = "true" ] || [ "${CLEAN_CILIUM_BPF_STATE}" = "true" ]; then rm -rf /sys/fs/bpf/tc/globals/cilium_* /var/run/cilium/bpffs/tc/globals/cilium_*; fi']
```

Comment:
Won't this fail for clean environments?

Reply:
I mean the "rm" of files that don't exist

Reply:
+1
```
rm -f ...
```

Reply:
I prefer not `rm -f`, we are root... What about `rm ... || true`?

Reply:
I wasn't sure about this note, but I don't think that `-e` is set in the shell where this is run, so simply running `rm ... ; ...` should be enough for it to continue. (I didn't test locally yet, but figured that if it can pass CI then it's not completely broken).

If you're wanting to hide extraneous errors that this might cause from init container logs, `-f` seems pretty reasonable; `|| true` won't have the same effect.